zebra-consensus/src/transaction.rs

```diff
@@ -55,6 +55,10 @@ where
 {
     // XXX: how should this struct be constructed?
     pub fn new(network: Network, script_verifier: script::Verifier<ZS>) -> Self {
+        // There was a discussion that instantiating the verifiers here or externally and passed as
+        // constructor parameters might help with testing:
+        // https://github.com/ZcashFoundation/zebra/issues/2390#issuecomment-870159482
+        //
         // let (spend_verifier, output_verifier, joinsplit_verifier) = todo!();
```

Comment:
I'd prefer to delete all of the references to the verifier fields in `struct Verifier` and `Verifier::new`, and turn that conversation into a ticket.

Otherwise, it looks like the implementation is incomplete, when it actually works just fine.

Reply:
Done in [e37e17e](https://github.com/ZcashFoundation/zebra/pull/2471/commits/e37e17ef3eeeaaa3edffb7beb51a9df47c1280e2). The tracking issue is #2473.